carapax-i18n/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# v0.1.0
+
+Initial commit
```

Comment:
Change this to:
```
# Changelog

## 0.1.0 (xx.xx.xxxx)

- First release.

```
in order to make changelog format same as in other projects

Reply:
I think we need rework changelog format